pkgs/by-name/ar/arrpc/package.nix

```diff
@@ -1,41 +1,31 @@
 { lib
 , buildNpmPackage
 , fetchFromGitHub
-, fetchpatch
-}:
-
-buildNpmPackage {
+}: buildNpmPackage {
```

Comment:
```suggestion
}: buildNpmPackage rec {
```

Reply:
I don't see a reason to use rec.

Reply:
But you can't use src.rev without rec.

Reply:
I can inherit it inside src, that's not exactly a problem. But I've went ahead with rec anyway.